parameter/ParameterMgr.cpp

```diff
@@ -1296,6 +1302,111 @@ CParameterMgr::CCommandHandler::CommandStatus CParameterMgr::getElementStructure
     return CCommandHandler::ESucceeded;
 }
 
+CParameterMgr::CCommandHandler::CommandStatus
+CParameterMgr::getElementBytesCommmandProcess(const IRemoteCommand& remoteCommand,
+                                              std::string& strResult)
+{
+    CElementLocator elementLocator(getSystemClass());
+
+    CElement* pLocatedElement = NULL;
+
+    if (!elementLocator.locate(remoteCommand.getArgument(0), &pLocatedElement, strResult)) {
+
+        return CCommandHandler::EFailed;
+    }
+
+    const CConfigurableElement* pConfigurableElement =
+            static_cast<CConfigurableElement*>(pLocatedElement);
+
+    // Prepare parameter access context for main blackboard.
+    // Notes:
+    //     - No need to handle output raw format and value space as Byte arrays are hexa formatted
+    //     - Pasing strResult to parameterAccessContext is only necessary wrt to constructor definition:
+    //       since it's a get type of access, no error may occur
+    CParameterAccessContext parameterAccessContext(strResult);
+    parameterAccessContext.setParameterBlackboard(_pMainParameterBlackboard);
+
+    // Get the settings
+    vector<uint8_t> bytes;
+    pConfigurableElement->getSettingsAsBytes(bytes, parameterAccessContext);
+
+    // Hexa formatting
+    std::ostringstream ostream;
+    ostream << std::hex << std::uppercase << std::setw(2) << std::setfill('0');
+
+    // Format bytes
+    for (auto byte : bytes) {
+
+        // Format
+        ostream << "0x" << byte;
```

Comment:
You are putting a `uint8_t` in a stream --> it will get formatted as a character (i.e. 100 -> "d" instead of "64")